doc/json-rpc.md

```diff
@@ -227,3 +234,115 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 ```
 
 ---
+
+### subscriptionChannel
+Open a socket on the node for JSON-RPC subscription requests
+
+##### Parameters:
+* `integer` - port to open for websocket subscription requests
+
+##### Results:
+* `string` - "addr", open websocket address
+* `string` - "path", unique key to use as websocket path
+
+##### Example:
+```bash
+// Request
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"method":"subscriptionChannel", "params":[3030]}' http://localhost:8899
+
+// Result
+{"jsonrpc":"2.0","result":{"addr":"127.0.0.1:3030","path":"BRbmMXn71cKfzXjFsmrTsWsXuQwbjXbwKdoRwVw1FRA3"},"id":1}
+```
+
+---
+
+### Subscription Websocket
+After opening a subscription socket with the `subscriptionChannel` JSON-RPC request method, submit subscription requests via websocket protocol
+Connect to the websocket at `ws://<ADDRESS>/<PATH>` returned from the request
+- Submit subscription requests to the websocket using the methods below
+- Multiple subscriptions may be active at once
+- The subscription-channel socket will close when client closes websocket. To create new subscriptions, send a new `subscriptionChannel` JSON-RPC reqest.
+
+---
+
+### accountSubscribe
+*alias: accountSub*
```

Comment:
IMO let's drop all the aliases, seems like it just adds confusion (ie, an option...)

Reply:
👌